packages/okta-auth-js/lib/builderUtil.js

```diff
@@ -91,8 +101,27 @@ function buildOktaAuth(OktaAuthBuilder) {
   };
 }
 
+function getUserAgent(args) {
+  var userAgent = args.userAgent;
+
+  if (!userAgent) {
+    return '';
+  }
+
+  if (userAgent.value) {
+    return userAgent.value;
+  }
+
+  if (userAgent.template) {
+    return userAgent.template.replace('$OKTA_AUTH_JS', `okta-auth-js/${SDK_VERSION}`);
```

Comment:
here there is a forward slash separating `okta-auth-js` from `SDK_VERSION` but the default implementation (server.js, line 32 has no slash)